housingbayarea/bloom
# fix: unit query separated from listing query

## Issue Overview
On staging environment (the hosted one) hitting certain listing pages was causing the backend to run out of memory and crash. This is a fix to split out the unit related data from the purely nonUnit listing data 


## Description
On staging environment (the hosted one) hitting certain listing pages was causing the backend to run out of memory and crash. This is a fix to split out the unit related data from the purely nonUnit listing data 

## How Can This Be Tested/Reviewed?
On local nothing should really change, you should be able to view the list of listings, the listing itself, applications, and on the public side see the listing as well 

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
